WordPress/gutenberg
# [RNMobile] Add drag & drop help guide in Help & Support screen

## What?


Update "Move blocks" section of "Help & Support" screen to incorporate a help guide for the drag & drop feature.

## Why?


Addresses https://github.com/wordpress-mobile/gutenberg-mobile/issues/4840.

## How?


The render function of `MoveBlocks` component has been updated to include further information about the drag & drop funcionality.

## Testing Instructions




1. Open a post/page.
2. Tap on the three dots button located at the top-right corner.
3. Tap on "Help & Support" button.
4. Observe that the "Move blocks" item has a different icon.
5. Tap on "Move blocks" item.
6. Observe that the screen contains information about the drag & drop feature.

**NOTE:** It would be great to test it in both light and dark modes.

## Screenshots or screencast 

**Help & Support screen _(note the change in the icon of Move blocks section)_:**
iOS (light)|iOS (dark)|Android (light)|Android (dark)
--|--|--|--
<img src=https://user-images.githubusercontent.com/14905380/168315999-e2b70010-b6a7-42c2-a872-1189e3c17304.png>|<img src=https://user-images.githubusercontent.com/14905380/168316015-a7e42e8b-295d-4ec2-bc93-4695ace98743.png>|<img src=https://user-images.githubusercontent.com/14905380/168316027-9f315830-6413-461b-b1a8-eb1bfa99abe4.jpg>|<img src=https://user-images.githubusercontent.com/14905380/168316042-64838165-dc04-47bc-b158-da57a37d0dc8.jpg>

**Move blocks section:**
iOS (light)|iOS (dark)|Android (light)|Android (dark)
--|--|--|--
<img src=https://user-images.githubusercontent.com/14905380/168104291-c8b31001-2feb-4394-9c9f-eee7ee6be121.png>|<img src=https://user-images.githubusercontent.com/14905380/168104313-70ad5ba9-68b7-4bf6-b121-4622787157c3.png>|<img src=https://user-images.githubusercontent.com/14905380/168104328-41abb5fc-5c6c-4906-869a-d6bdf66f1a1a.jpg>|<img src=https://user-images.githubusercontent.com/14905380/168104340-a8d53b67-fb1d-4b94-8252-59d4eba74d22.jpg>
